src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedAssemblyReference.xaml

```diff
@@ -135,7 +135,7 @@
     <BoolProperty.DataSource>
       <DataSource HasConfigurationCondition="False"
                   ItemType="Reference"
-                  Persistence="AssemblyReference"
+                  Persistence="ProjectFile"
```

Comment:
I think this property-level data source was copied from the rule-level one, and the `Persistence` property not updated.